error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -624,4 +625,17 @@ R after(
       return stream.flatMap(mapper).collect(collector);
     }
   }
+
+  static final class StreamsConcat<T> {
+    @BeforeTemplate
+    @SuppressWarnings("StreamOfArray")
+    Stream<T> before(@Repeated Stream<T> stream) {
+      return Stream.of(Refaster.asVarargs(stream)).flatMap(Function.identity());
```

Comment:
It won't have functional impact, but we generally statically import `Function.identity()`.

Something that is functionally relevant, is that _for now_, we don't have a rule that favours `identity()` over `v -> v` or vice versa. (There are technical reasons for that.) So let's also capture that case.